package.json

```diff
@@ -22,6 +24,7 @@
     "test": "watai test && watai test/bootstrap-compat",
     "precompile": "rimraf dist/*",
     "compile": "webpack --optimize-minimize",
-    "compile-continuously": "webpack --watch"
+    "compile-continuously": "webpack --watch",
+    "dev": "webpack-dev-server"
```

Comment:
Works now that the dependency has been added, but the latest `main.css` rules don't apply (no `max-width`, `line-height`…).